data-node/src/main/java/org/graylog/datanode/Configuration.java

```diff
@@ -158,23 +176,36 @@ public class Configuration {
     @Parameter(value = HTTP_CERTIFICATE_PASSWORD_PROPERTY)
     private String datanodeHttpCertificatePassword;
 
-    @Documentation("You MUST set a secret to secure/pepper the stored user passwords here. Use at least 64 characters." +
-            "Generate one by using for example: pwgen -N 1 -s 96 \n" +
-            "ATTENTION: This value must be the same on all Graylog and Datanode nodes in the cluster. " +
-            "Changing this value after installation will render all user sessions and encrypted values in the database invalid. (e.g. encrypted access tokens)")
+    @Documentation("""
+            You MUST set a secret to secure/pepper the stored user passwords here. Use at least 64 characters.
+            Generate one by using for example: pwgen -N 1 -s 96
+            ATTENTION: This value must be the same on all Graylog and Datanode nodes in the cluster.
+            Changing this value after installation will render all user sessions and encrypted values
+            in the database invalid. (e.g. encrypted access tokens)
+            """)
     @Parameter(value = "password_secret", required = true, validators = StringNotBlankValidator.class)
     private String passwordSecret;
 
-    @Documentation("communication between Graylog and OpenSearch is secured by JWT. This configuration defines interval between token regenerations.")
+    @DocumentationSection(heading = "OpenSearch JWT token usage",description = """
+            communication between Graylog and OpenSearch is secured by JWT. These are the defaults used for the token usage
+            adjust them, if you have special needs.
+            """)
+    @Documentation(value = "This configuration defines interval between token regenerations.")
     @Parameter(value = "indexer_jwt_auth_token_caching_duration")
     Duration indexerJwtAuthTokenCachingDuration = Duration.seconds(60);
 
-    @Documentation("communication between Graylog and OpenSearch is secured by JWT. This configuration defines validity interval of JWT tokens.")
+    @DocumentationSection(heading = "OpenSearch JWT token usage",description = """
+            communication between Graylog and OpenSearch is secured by JWT. These are the defaults used for the token usage
```

Comment:
I see it as an ugly workaround and not a proper solution, which is dividing the configuration sections into more smaller config classes, each with one section heading and description on a class level. We can try to make it for now less ugly and try to extract and reuse the heading and section, but from my POV it will be still an ugly hack. Our documentation should follow the structure of the classes and then the problem will disappear. That's why I think the repetition doesn't deserve much attention, as it solves wrong problem. 

As an additional benefit, we'll get smaller config classes, with better naming, not just one big Configuration class containing everything.